Doc/library/asyncio-eventloops.rst

```diff
@@ -165,7 +165,7 @@ An event loop policy must implement the following interface:
 
    .. method:: get_event_loop()
 
-      Get the event loop for the current context.
+      Get the event loop for the current coroutine.
```

Comment:
See the documentation of [`AbstractEventLoop.run_forever()`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.AbstractEventLoop.run_forever) for an example. In this case, it says, "Changed in version 3.5.1."